src/include/com/amazonaws/kinesis/video/common/Include.h

```diff
@@ -84,6 +74,18 @@ extern "C" {
 #define STATUS_TIMESTAMP_STRING_UNRECOGNIZED_FORMAT     STATUS_COMMON_BASE + 0x00000005
 /*!@} */
 
+/**
+ * Macro for checking whether the status code should be retried by the continuous retry logic
+ */
+#define IS_RETRIABLE_COMMON_LIB_ERROR(error)                                                                                                         \
+    ((error) == STATUS_INVALID_API_CALL_RETURN_JSON || (error) == STATUS_CURL_INIT_FAILED || (error) == STATUS_CURL_LIBRARY_INIT_FAILED ||           \
+     (error) == STATUS_HMAC_GENERATION_ERROR || (error) == STATUS_CURL_PERFORM_FAILED || (error) == STATUS_IOT_INVALID_RESPONSE_LENGTH ||            \
+     (error) == STATUS_IOT_NULL_AWS_CREDS || (error) == STATUS_IOT_INVALID_URI_LEN || (error) == STATUS_IOT_EXPIRATION_OCCURS_IN_PAST ||                   \

```

Comment:
Can we perhaps fix the `\`?

Or, if this is just a github display issue, you can ignore

Reply:
I think it is a github thing. Will wait to see if clang format reports anything. If not, leaving it as is.